std_msgs/Header header
geometry_msgs/Transform origin_transform
float64 sdf_cell_size
float32 OOB_value
geometry_msgs/Vector3 dimensions
uint8[] data
